egograph/backend/infrastructure/logging/sanitizers.py

```
@@ -0,0 +1,71 @@
+    - record.exc_text（トレースバック）が存在すればサニタイズする
+    - すべてのレコードを通過させる（ドロップしない）
+    """
+
+    def filter(self, record: logging.LogRecord) -> bool:
+        """ログレコードをサニタイズして通過させる。
+
+        Args:
+            record: フィルタ対象のログレコード。
+
+        Returns:
+            常に True（レコードをドロップしない）。
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)